app/helpers/controllers/validateOrderItemFormBulk.test.js

```diff
@@ -182,6 +182,46 @@ describe('validateOrderItemFormBulk', () => {
       expect(errors).toEqual([priceGreaterThanListPrice]);
     });
 
+    it('should not error if price is greater than the list price but less than the orignal price', () => {
+      getSelectedPriceManifest.getSelectedPriceManifest.mockReturnValue(selectedPriceManifest);
+      const data = {
```

Comment:
This is confusing – it should error if the entered price is greater than the list price. I'm not sure what original price is in this context.

There are only two prices that we should be concerned with: the entered price and the list price.